enterprise/frontend/src/metabase-enterprise/moderation/components/ModerationReviewBanner/ModerationReviewBanner.tsx

```diff
@@ -1,52 +1,44 @@
-import PropTypes from "prop-types";
 import { connect } from "react-redux";
 import _ from "underscore";
 
 import Users from "metabase/entities/users";
-import { color, alpha } from "metabase/lib/colors";
+import { alpha, color } from "metabase/lib/colors";
 import { getRelativeTime } from "metabase/lib/time";
 import { getUser } from "metabase/selectors/user";
 import { Icon } from "metabase/ui";
 import {
-  getTextForReviewBanner,
   getIconForReview,
+  getTextForReviewBanner,
 } from "metabase-enterprise/moderation/service";
+import type { ModerationReview, User } from "metabase-types/api";
 
 import {
   Container,
   Text,
-  Time,
   TextContainer,
+  Time,
 } from "./ModerationReviewBanner.styled";
 
 const ICON_BUTTON_SIZE = 16;
 
-const mapStateToProps = (state, props) => ({
+const mapStateToProps = (state: any, _props: any) => ({
```

Comment:
we have a type for  state

```ts
import type { State } from "metabase-types/store";
```

I think you should also be able to type props here pretty easily, right?, or just omit it entirely since you're not using it?